src/test/java/org/cactoos/io/TempFolderTest.java

```diff
@@ -38,13 +38,11 @@
 public final class TempFolderTest {
 
     @Test
-    public void createsDirectory() {
+    public void createsDirectory() throws Exception {
+        final File dir = new TempFolder().value().toFile();
```

Comment:
@victornoel `TempFolder` is meant to be used in a `try-with-resources` block

Reply:
@llorllale ok... I didn't change anything related to that, but I will add it then